src/Symfony/Component/HttpKernel/Controller/ArgumentValueResolverInterface.php

```diff
@@ -37,7 +37,7 @@ public function supports(Request $request, ArgumentMetadata $argument);
      * @param Request          $request
      * @param ArgumentMetadata $argument
      *
-     * @return \Generator
+     * @return iterable
```

Comment:
#31996 added some iterable return types into implementations. This would actually be invalid if we had real return types as you cannot widen the return types (even in PHP 7.4). So we either need to make this iterable which I think makes sense as the calling code does not need a Generator at all. Or we need to change some types again in master.